Sources/ArcGISToolkit/Common/AttachmentsFeatureElementView.swift

```diff
@@ -67,18 +67,33 @@ struct AttachmentsFeatureElementView: View {
     @State private var isExpanded = true
     
     var body: some View {
-        Group {
+        Section {
```

Comment:
@mhdostal @dfeinzimer 

I had to make this change compared to the previous change I had because there was some strange behavior where the attachments weren't getting loaded and a cancellation error was getting thrown. So if I wrap this whole view in a Section that solves the problem and maybe this is the way SwiftUI expects it to be.

The main thing I wanted to point out was `AttachmentsFeatureElementView` is used for feature forms and popups so I made edits here to prevent any new obvious changes happening to popups.

Reply:
I wonder if the unexpected cancellation is related to the Form content being lazy loaded in some way.

Reply:
Yeah that was one of the things Phil and I discussed. It could be.